DemoApp/DemoApp/Views/TMInputPage.xaml.cs

```diff
@@ -1,6 +1,6 @@
 using DemoApp.ViewModels;
 using System.Text.RegularExpressions;
-using Trimble.Modus.Components;
+using Trimble.Modus.Components.Controls;
```

Comment:
I see you have moved the controls to the `Controls` folder, but please keep the namespace as it is. Users should use just `Trimble.Modus.Components`.